### DISCUSSION TITLE
Moving to an A1111 fork

### DISCUSSION THREAD

**Discussion by Brown-Michael** [TheLastBen/fast-stable-diffusion#2004] [General]
Would this ever happen? Auto hasn't been updating a lot recently and there are a bunch of new things that are being added to some forks (pytorch 2.0 and controlnet1.1 for example)

**Comment by TheLastBen** [comment]
Colab is already pytorch 2

**Reply by Brown-Michael** [reply]
Didn't know, thank you for the quick reply
